app/src/modules/settings/routes/extensions/components/extension-item.vue

```diff
@@ -40,27 +42,28 @@ const toggleEnabled = async () => {
 		await api.patch(endpoint, { meta: { enabled: !props.extension.meta.enabled } });
 	} finally {
 		changingEnabledState.value = false;
-		emit('refresh');
+		emit('refresh', extensionType);
 	}
 };
 </script>
 
 <template>
-	<v-list-item block :class="{ disabled: !extension.meta.enabled }">
+	<v-list-item block :class="{ disabled: devMode ? false : !extension.meta.enabled }">
 		<v-list-item-icon v-tooltip="t(`extension_${type}`)"><v-icon :name="icon" small /></v-list-item-icon>
 		<v-list-item-content class="monospace">{{ extension.name }}</v-list-item-content>
 		<v-chip v-if="extension.schema?.version" class="version" small>{{ extension.schema.version }}</v-chip>
 
 		<template v-if="extension.schema?.type !== 'bundle'">
 			<v-progress-circular v-if="changingEnabledState" indeterminate />
-			<v-chip v-else class="state" :class="{ enabled: extension.meta.enabled }" small>
-				{{ extension.meta.enabled ? t('enabled') : t('disabled') }}
+			<v-chip v-else class="state" :class="{ enabled: devMode ? true : extension.meta.enabled }" small>
```

Comment:
Could we add a tooltip hint here à la "App extensions are always enabled in dev mode" or "App extensions cannot be disabled in dev mode" to make it a bit clearer?

<img width="155" src="https://github.com/directus/directus/assets/5363448/71b86cfe-f4a5-47d1-9331-c8d21297a784">